src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -154,6 +154,24 @@ internal ReadOnlySpan(ref T ptr, int length)
 #endif
         }
 
+        public T this[Index index]
```

Comment:
Should this return `ref readonly` like the regular indexer?

Reply:
I following up with @agocke if this will be ok from the compiler side and then change it as you suggested.

Reply:
I confirmed that would be ok, so I'll go ahead and apply your suggestion.